# Assembly shipment contents message
# This structure contains the information about an assembly shipment being submitted

# The ID of the shipping_box or tray the shipment was detected in
string briefcase_id

# Collection of products
DetectedProduct[] products